edge/pkg/edged/edged.go

```diff
@@ -1009,25 +1009,23 @@ func (e *edged) podRemoveWorkerRun(consumers int) {
 
 func (e *edged) consumePodAddition(namespacedName *types.NamespacedName) error {
 	podName := namespacedName.Name
-	klog.Infof("start to consume added pod [%s]", podName)
+	klog.V(4).Infof("start to consume added pod [%s]", podName)
```

Comment:
We can remain the klog.Infof here, always need to check whether the pods are handled by edged.